pkg/storage/fs/owncloud/owncloud.go

```diff
@@ -77,6 +77,9 @@ const (
 	checksumPrefix    string = ocPrefix + "cs."
 	checksumsKey      string = "http://owncloud.org/ns/checksums"
 	favoriteKey       string = "http://owncloud.org/ns/favorite"
+
+	spaceTypeAny = "*"
+	// spaceIDAny   = "*"
```

Comment:
drop this line

Reply:
Done in https://github.com/cs3org/reva/pull/2383